src/content/docs/design/customize-with-code/styling-with-css.mdx

```diff
@@ -18,6 +18,12 @@ For example, you can:
 - Set the primary button’s background color with `--kinde-button-primary-background-color`.
 - Set the color of a form field’s invalid message with `--kinde-control-associated-text-invalid-message-color` (or use `--kinde-shared-color-invalid` for a more global approach).
 
+## Storing your CSS and other asset files
+
+We know you will want to use assets like stylesheets, fonts and images. Kinde does not currently offer hosting for these static assets and you will need to host them yourself.
+
+We recommend storing them on servers that share the same domain as your custom domain. For example, if you are using the custom domain auth.myapp.com you could host your external fonts, images and CSS files at assets.myapp.com and they would be accessible in your code.
```

Comment:
@clairekinde11 "auth.myapp.com" and "assets.myapp.com" needs backticks (marked as "code") like in `src/content/docs/design/customize-with-code/manage-design-assets.mdx`

* `auth.myapp.com`
* `assets.myapp.com`